### ISSUE TITLE
Link to how people can submit issues for the website

### ISSUE THREAD

**Issue by pritianka** [cncf/cncf.io#366]
Use case: Often times when perusing the site I come across things we could improve. I am sure this happens to other visitors as well. It would be nice to link them to our issue board with an issue template for folks to share feedback/ideas. GitLab does this well on their site but they ask people to directly contribute. We could use a similar placement to link to our issues. Thoughts?
<img width="1612" alt="GL_open_improvements" src="https://user-images.githubusercontent.com/66081113/103466744-6f6fce80-4cfc-11eb-95b6-bb0b88aed630.png">

cc @cjyabraham @ccraigross

**Comment by cjyabraham** [comment]
Sounds good.  How about a link on [our Contact page](https://www.cncf.io/about/contact/):

"For comments on this website or to report an issue please [create an issue on our GitHub repo](https://github.com/cncf/cncf.io/issues/new)."

Perhaps not much more is needed than that?  I could setup a template for the github new issue form but, given the audience, it may not be necessary?

**Comment by pritianka** [comment]
I am torn between having it at the bottom of every page vs on the contact page. The pro of the former is that people can click to submit feedback as they go through the site. The con is that it may be overkill for an audience not used to it. The con for the contact page is that someone would need to think a lot about wanting to submit feedback and then finding the link on the contact page.

**Comment by cjyabraham** [comment]
My vote would be for the Contact page for now, in addition to cleaning up the other text on that page.  I think we already have so much going on in the footer it would be hard to make it stand out.

**Comment by thetwopct** [comment]
I like the way GitLab have it, but it's slightly easier for them to offer contributing/feedback facility as their entire public-facing website code and content is public and editable, whereas our content is private in a database.

Adding an issue request to the footer of each page makes sense to me, I don't feel like it makes it too busy. We can setup the link to open an issue with some information already filled in (such as the URL it came from and assign it to the web project) - [Example](https://github.com/cncf/cncf.io/issues/new?title=Your+issue&projects=cncf/cncf.io/1&body=From+URL%3A+https://www.cncf.io/about/contact/).

How it could look:

![2021-01-03-125810@2x](https://user-images.githubusercontent.com/10615884/103479124-60a01f00-4dc3-11eb-88fa-feff9ec38993.jpg)

I've added a "Contribute" link too (linking to the repo) - not many Enterprise-level WordPress codebases are open source, so promoting it a bit more won't hurt!

**Comment by cjyabraham** [comment]
One final option is we have both: as James has suggested in the footer and also on the Contact page.

I have mixed feelings, however, on requesting people contribute.  People have tried to contribute to our site in the past who are clearly not well-suited to the task and we have to go out of our way to politely steer them away.

**Comment by cjyabraham** [comment]
Any final thoughts @pritianka ?

**Comment by thetwopct** [comment]
I've added this to the footer so it will appear on all pages. [Preview PR](https://pr-384-cncfci.pantheonsite.io/#newsletter).

![2021-02-25-225749](https://user-images.githubusercontent.com/10615884/109230522-ed140000-77bc-11eb-95bf-f749adc9d7fb.jpg)

I added "... on GitHub" to the text as if not signed in to GitHub it will dump the user on a GitHub login page, so this may be a surprise to them and I felt this explained it a bit more.
